input/fsh/SD_Medications.fsh

```diff
@@ -21,11 +21,14 @@ Description:    "An episode of medication administration for a patient whose con
 RuleSet: MedicationResourcesRS
 * ^extension[FMM].valueInteger = 3
 * obeys mcode-reason-required
+* obeys termination-reason-code-invariant 
+* obeys termination-reason-invariant 
 * subject only Reference(CancerPatient)
 * subject ^definition = "The patient receiving the medication."
 * extension contains
     ProcedureIntent named treatmentIntent 0..1 MS and
     TreatmentTerminationReason named terminationReason 0..* MS
```

Comment:
Users can include the TreatmentTerminationReason extensions without it being explicitly in the profile.